packages/thirdweb/src/exports/utils.ts

```diff
@@ -122,4 +122,6 @@ export {
 // ------------------------------------------------
 export { encodeAbiParameters } from "../utils/abi/encodeAbiParameters.js";
 
+// Useful helpers
 export { setThirdwebDomains } from "../utils/domains.js";
+export { resolvePromisedValue } from "../utils/promise/resolve-promised-value.js";
```

Comment:
check that it has proper jsdoc if you're exporting it (not marked as `@internal`)